README.md

```diff
@@ -12,41 +12,28 @@ This npm [mono-repo][mono-repo] contains editor related tooling for [SAPUI5][ui5
 
 It currently contains:
 
-The [UI5 language assistant](./packages/vscode-ui5-language-assistant) VSCode Extension.
-
-The following packages:
-
-- [![npm-language-server][npm-language-server-image]][npm-language-server-url] [@ui5-language-assistant/language-server](./packages/language-server) UI5 Language Server.
-- [![npm-logic-utils][npm-logic-utils-image]][npm-logic-utils-url] [@ui5-language-assistant/logic-utils](./packages/logic-utils) Common shared logic and data structures related utilities used in other packages in this mono-repo.
-- [![npm-semantic-model][npm-semantic-model-image]][npm-semantic-model-url] [@ui5-language-assistant/semantic-model](./packages/semantic-model) Data structure and related utils to represent the UI5 metadata.
-- [![npm-semantic-model-types][npm-semantic-model-types-image]][npm-semantic-model-types-url] [@ui5-language-assistant/semantic-model-types](./packages/semantic-model-types) Type signatures for the semantic model.
-- [![npm-settings][npm-settings-image]][npm-settings-url] [@ui5-language-assistant/settings](./packages/settings) Settings used in other packages in this mono-repo.
-- [![npm-user-facing-text][npm-user-facing-text-image]][npm-user-facing-text-url] [@ui5-language-assistant/user-facing-text](./packages/user-facing-text) User facing text used in other packages in this mono-repo.
-- [![npm-xml-views-completion][npm-xml-views-completion-image]][npm-xml-views-completion-url] [@ui5-language-assistant/xml-views-completion](./packages/xml-views-completion) Logic for UI5 XML views code assist.
-- [![npm-xml-views-tooltip][npm-xml-views-tooltip-image]][npm-xml-views-tooltip-url] [@ui5-language-assistant/xml-views-tooltip](./packages/xml-views-tooltip) Logic for UI5 XML views hover tooltip.
-- [![npm-xml-views-validation][npm-xml-views-validation-image]][npm-xml-views-validation-url] [@ui5-language-assistant/xml-views-validation](./packages/xml-views-validation) Logic for UI5 XML views semantic validations.
-- [![npm-xml-views-quick-fix][npm-xml-views-quick-fix-image]][npm-xml-views-quick-fix-url] [@ui5-language-assistant/xml-views-quick-fix](./packages/xml-views-quick-fix) Logic for UI5 XML-Views quick-fixes.
+- The [UI5 language assistant](./packages/vscode-ui5-language-assistant) VSCode Extension consisting of the following packages:
+
+  - [@ui5-language-assistant/language-server](./packages/language-server) UI5 Language Server.
+  - [@ui5-language-assistant/context](./packages/context) Common shared data model container (UI5 API loaded version cache, project information, ODATA services, manifest information, etc.) with utilities to load and parse the data used in other packages in this mono-repo.
+  - [@ui5-language-assistant/fe](./packages/context) UI5 Language Server extension to support completion and diagnostics of ODATA annotations related data.
```

Comment:
This seems a bit too wide a scope, since it should only include features that are relevant to SAP Fiori elements building blocks. Maybe something like this:
```
UI5 Language Server extension to support SAP Fiori elements building blocks.
```